app/Http/Controllers/CompetitionsController.php

```diff
@@ -37,18 +37,6 @@ public function __construct(UserRepositoryContract $repository, Manager $manager
     }
 
     /**
-     * Display a listing of the resource.
-     *
-     * @return \Illuminate\Http\Response
-     */
-    public function index()
```

Comment:
Cool. I'm just gonna leave it out for now then. It doesn't take long to add back in and then at least when it's added back in it's cause we found an need for it and it's not an extraneous method/file forgotten in the annals of code repos :hourglass: